package.json

```diff
@@ -115,5 +115,8 @@
     "svgo": "0.6.1",
     "webpack": "1.12.12",
     "webpack-dev-server": "1.14.1"
+  },
+  "dependencies": {
+    "eidolon": "^1.2.0"
```

Comment:
Nice catch. Does this mean Attributes Kit can never be `npm install`ed and used server-side in Node?